msteams-platform/m365-apps/actions-in-m365-quick-start-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Actions in Microsoft 365 quick start guide
+description: In this article, learn how to create Actions in Microsoft 365, function of Actions and its use cases.
+ms.date: 12/8/2023
+ms.author: mosdevdocs
+author: v-preethah
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365 quick start guide
+
+> [!NOTE]
+>
+> * Actions are available in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+> * Actions are supported for Microsoft 365 (Office) app for web and desktop.
+
+Get started with Actions in Microsoft 365, which helps integrate an app into the user's workflow by enabling easy discoverability and seamless interaction with their content.
+
+## Prerequisites
+
+Before you get started, ensure that you install the following tools:
+
+| &nbsp; | Install | For using... |
+| --- | --- | --- |
+| &nbsp; | [Node.js](https://nodejs.org/en/download/) | A JavaScript runtime environment. For more information, see [Node.js version compatibility table for project type](~/toolkit/build-environments.md#nodejs-version-compatibility-table-for-project-type).|
+| &nbsp; | [Microsoft Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/) | A browser with developer tools. |
+| &nbsp; | [Visual Studio Code](https://code.visualstudio.com/download) | Visual Studio Code is a lightweight but powerful source code editor, which comes with built-in support for JavaScript, TypeScript, Node.js, and SharePoint Framework (SPFx) build environments. Use the latest version. |
+| &nbsp; | [Teams Toolkit](../toolkit/install-Teams-Toolkit.md) | A Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use the latest version. |
+
+### Build and run the sample app
+
```

Comment:
Maybe add a little description that this instruction tells you how to build and run the sample app in your local environment, try with sideload the app to your own dev account.